SofaBenchmarkScenes/src/SofaBenchmarkScenes/BenchScene.h

```diff
@@ -42,7 +43,6 @@ void BM_Scene_bench_SimulationFactor(benchmark::State& state)
             {
                 simu->animate(root.get(), TScene::dt);
             }
-            state.PauseTiming();
```

Comment:
removing this PauseTiming() will make the unload(root) counts in the timings no ? 🤔

Reply:
Yes, but here is the weird part: with the pause, it doubles the duration of the benchmark. Without it, it is more accurate. I cannot explain it.